src/Business/Camera/Fetching/Models/PCamera.cs

```diff
@@ -0,0 +1,9 @@
+namespace Business.Camera.Fetching.Models
+{
+    using System.Collections.Generic;
+
+    public class PCamera
+    {
+        public List<string> Ids { get; set; } = new List<string>();
```

Comment:
Should remove the default value for this? If it's null, we should skip the filtering. Otherwise we won't be able to fetch all the existent cameras.

Reply:
Should we also add the pagination parameter here?

Reply:
Should we also convert the `Ids` property to be a nullable one?
https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references

Reply:
Done